OfficeAddin/01 Building Add-ins for Microsoft Word/Demos/README.md

```diff
@@ -4,13 +4,13 @@ This folder contains demos for the Building Office Add-ins for Word module.
 
 ## Demos
 
-- [Text and Formatting](./Demos/01%20Text%20and%20Formatting)
-- [Images, HTML and Tables](./Demos/02%20Images%20HTML%20and%20Tables)
-- [Content Controls](./Demos/03%20Content%20Controls)
+- [Text and Formatting](./01%20Text%20and%20Formatting)
```

Comment:
@andrewconnell Won't this break the link to the demos folder?

Reply:
Are you sure you were on the correct branch when checking the links? The `/demos` part was removed from the URLs as it was not working, but if you tested by mistake from a different branch, they wouldn't work (I made the same mistake when testing myself).

If you look at the branch where I submitted these changes, they are working: https://github.com/andrewconnell/TrainingContent/tree/fy2019q1-qarefresh-officeaddin-01/OfficeAddin/01%20Building%20Add-ins%20for%20Microsoft%20Word/Demos

Reply:
You're correct. I was checking in the wrong readme file. My bad. Okay this is good. Thanks!